CHANGELOG.md

```diff
@@ -3,12 +3,16 @@
 All notable changes to this project are documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## 0.2.0 (2015-06-12)
 
 ### Added
 
+  * --data option for extra data to be added to the message payload
+
 ### Changed
 
+  * option --broker renamed to --broker-list and accpeting a comma separated list of brokers
```

Comment:
small typo and spelling is "comma-separated"

you should also use statements in simple tense, as in "option --broker renamed to --broker-list and accepts comma-separated list of brokers"